packages/wells/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "private": true,
+  "private": false,
```

Comment:
I have set ` "publishConfig": {"access": "public"}` and `"private": false` in the package.json file. Is this the right way to publish the package?